### PR TITLE
T10- Reverse a given number.

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter number: ");
+         int a = int.Parse (Console.ReadLine ());
+         int num = a;
+         int reversed_num = 0;
+         while (a > 0) {
+            int rem = a % 10;
```
Comment (MetaNaveen) [inline Program.cs:9]: rem is used only once. So could be used directly in the expression statement.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+            reversed_num = reversed_num * 10 + rem;
+            a = a / 10;
```
Comment (MetaNaveen) [inline Program.cs:11]: shorthand could be used. a /= 10;

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+         }
+         Console.WriteLine ($"Reversed number {reversed_num}.");
+         if (num == reversed_num) {
+            Console.WriteLine ($"{num} is palindrome number.");
+         } else
+            Console.WriteLine ($"{num} is not a palindrome number.");
```
Comment (MetaNaveen) [inline Program.cs:17]: Why curly braces for single line statments? That too, for 'if' but not for 'else'?

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

**Comment by MetaNaveen** [comment]
Working as expected.

**Final Notes:**
Minor formatting issues.

**Comment by Priyanka15-12** [comment]
Formatting issue fixed.

**Pull request closed without merging**